### PR TITLE
Update psc list screen to version 9 of prototype

### PR DESCRIPTION
**Pull request by AlisonGriffiths**
* Modified psc list web page content
* Updated locale files
* Added temporary fix to routing issue by including psc id as query param (to be removed in later story)
* Fixed tests by adding the new query param

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/views/router_views/individualPscList/individualPscList.njk [hunk 1]
```diff
@@ -1,35 +1,136 @@
...
+    }
+  ]
+}) }}
+
+{% set pscNotFoundDetails %}
+        <p class="govuk-body">{{ i18n.ind_psc_list_not_found_details_description }}</p>
+        <p class="govuk-body">{{ i18n.ind_psc_list_not_found_details_description2 }}</p>
+        <p class="govuk-body">{{ i18n.ind_psc_list_not_found_details_description3 }}{{ dsrEmailAddress }}
+            {{ i18n.ind_psc_list_not_found_details_description4 }}{{ dsrPhoneNumber }}
+            {{ i18n.ind_psc_list_not_found_details_description5 }}</p>
```
Comment (eedwards0) [inline src/views/router_views/individualPscList/individualPscList.njk:124]: Just a note on something I've seen before, but not found a solution(!): extra white space follows a variable inserted within the JSON text e.g. after `{{ dsrPhoneNumber }}`.

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Pull request merged**